src/calculateRentalCost.js

```diff
@@ -4,7 +4,16 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRate = 40;
+  let totalCost = days * dailyRate;
+
+  if (days >= 7) {
+    totalCost -= 50;
+  } else if (days >= 3) {
+    totalCost -= 20;
```

Comment:
Replace the literal '3' with a named constant to explain what this value represents, such as 'SHORT_TERM_RENTAL_DAYS'. Do the same for the discount value '20', using a name like 'SHORT_TERM_DISCOUNT'.